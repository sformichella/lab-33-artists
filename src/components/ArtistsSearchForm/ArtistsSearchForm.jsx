import React from 'react';

export default function ArtistsSearchForm() {
  return (
    <div>
      
    </div>
  );
}
