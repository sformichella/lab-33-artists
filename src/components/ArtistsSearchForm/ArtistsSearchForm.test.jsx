import React from 'react';
import { render, cleanup } from '@testing-library/react';
import ArtistsSearchForm from './ArtistsSearchForm';

describe('ArtistsSearchForm component', () => {
  afterEach(() => cleanup());
  it('renders ArtistsSearchForm', () => {
    const { asFragment } = render(<ArtistsSearchForm />);
    expect(asFragment()).toMatchSnapshot();
  });
});
