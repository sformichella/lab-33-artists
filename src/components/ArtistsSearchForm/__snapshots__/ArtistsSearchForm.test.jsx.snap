// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistsSearchForm component renders ArtistsSearchForm 1`] = `
<DocumentFragment>
  <form>
    <label
      for="artist-search"
    >
      Search: 
    </label>
    <input
      id="artist-search"
      type="text"
    />
    <button
      type="submit"
    >
      Search
    </button>
  </form>
</DocumentFragment>
`;
